tests/backup/backup_helper.go

```diff
@@ -1245,7 +1263,202 @@ func backupSuccessCheck(backupName string, orgID string, retryDuration time.Dura
 	return nil
 }
 
-// restoreSuccessCheck inspects restore task
+// ValidateBackup validates a backup's spec's objects (resources) and volumes. This function must be called after switching to the context on which `scheduledAppContexts` exists. Cluster level resources aren't validated.
+func ValidateBackup(ctx context.Context, backupName string, orgID string, scheduledAppContexts []*scheduler.Context) error {
+	log.InfoD("Validating backup [%s] in org [%s]", backupName, orgID)
+
+	log.InfoD("Obtaining backup info for backup [%s]", backupName)
+	backupDriver := Inst().Backup
+	backupUid, err := backupDriver.GetBackupUID(ctx, backupName, orgID)
+	if err != nil {
+		return fmt.Errorf("GetBackupUID Err: %v", err)
+	}
+	backupInspectRequest := &api.BackupInspectRequest{
+		Name:  backupName,
+		Uid:   backupUid,
+		OrgId: orgID,
+	}
+	backupInspectResponse, err := backupDriver.InspectBackup(ctx, backupInspectRequest)
+	if err != nil {
+		return fmt.Errorf("InspectBackup Err: %v", err)
+	}
+
+	backupStatus := backupInspectResponse.GetBackup().GetStatus().Status
```

Comment:
I would suggest to declare a variable and assign it as follows to avoid repeating this multiple times `backupInspectResponse.GetBackup()`

```
backupObj := backupInspectResponse.GetBackup()
```

Reply:
✅